app/views/physical_server/show.html.haml

```diff
@@ -1,5 +1,12 @@
+- @angular_form = true
```

Comment:
I see this on places and I wonder.

This is to skip the content of `app/views/layouts/_x_edit_buttons.html.haml` or does it have some more meaning?

I am wondering on it's purpose since setting variables inside a view. 

We are actually counting on that it leaks somewhere. Help me understant why is this needed maybe I can help to figure out some better way to do it.